src/powershell/install.sh

```diff
@@ -162,6 +163,12 @@ if [ ${#POWERSHELL_MODULES[@]} -gt 0 ]; then
     done
 fi
 
+# If URL for powershell profile is provided, download it to '/opt/microsoft/powershell/7/profile.ps1'
+if [ -n "$POWERSHELL_PROFILE_URL" ]; then
+    echo "Downloading PowerShell Profile from: $POWERSHELL_PROFILE_URL"
+    curl -sSL -o "/opt/microsoft/powershell/7/profile.ps1" "$POWERSHELL_PROFILE_URL"
```

Comment:
> /opt/microsoft/powershell/7/profile.ps1

Looks like ^ would work for Powershell version 7, however, how about other versions? Like 5.1, doesn't seem like `/opt/microsoft/powershell/7/profile.ps1` would work. See https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_profiles?view=powershell-5.1

Reply:
PowerShell 5.1 is not available for Linux and PowerShell 6 is deprecated. Only 7.* is [currently supported](https://learn.microsoft.com/en-us/powershell/scripting/install/powershell-support-lifecycle?view=powershell-7.4), so this should cover most use cases for now. We can expand it later to support other version as well, if needed.